src/ngraph/op/experimental/quantized_conv_bias.cpp

```diff
@@ -192,9 +193,9 @@ op::QuantizedConvolutionBiasSignedAdd::QuantizedConvolutionBiasSignedAdd(
     // TODO: call ngraph util
     // util::validate_convbias_shapes(data_batch_shape, filters_shape, bias->get_shape());
 
-    auto output_et = with_relu ? element::u8 : element::i8;
+    assert(with_relu == true);
```

Comment:
Can you please throw an `ngraph_error` or `AssertionFailure` or some ngraph exception. In the bridge we catch ngraph-errors and print some error messages